autotest/gdrivers/heif.py

```diff
@@ -117,6 +117,248 @@ def test_heif_rgba_16bit():
     assert ds.GetRasterBand(1).DataType == gdal.GDT_UInt16
 
 
+def _has_tiling_support():
+    drv = gdal.GetDriverByName("HEIF")
+    return drv and drv.GetMetadataItem("HEIF_SUPPORTS_TILES")
+
+
+def test_heif_tiled():
+    if not _has_tiling_support():
+        pytest.skip()
+
+    ds = gdal.Open("data/heif/uncompressed_comp_RGB_tiled.heif")
+    assert ds
+    assert ds.RasterXSize == 30
+    assert ds.RasterYSize == 20
+    assert ds.RasterCount == 3
+    assert ds.GetRasterBand(1).DataType == gdal.GDT_Byte
+    assert ds.GetRasterBand(1).GetBlockSize() == [15, 5]
+    assert ds.GetRasterBand(2).GetBlockSize() == [15, 5]
+    assert ds.GetRasterBand(3).GetBlockSize() == [15, 5]
+    assert (
```

Comment:
GDAL builds don't strictly require numpy / gdal_array support so:
```suggestion
    pytest.importorskip("osgeo.gdal_array")
    assert (
```

Reply:
Thanks for this.